tools/cassandra/handler.go

```diff
@@ -75,7 +75,7 @@ func verifyCompatibleVersion(
 	// However, this file will be refactor to support NoSQL soon. After the refactoring, cycle dependency issue
 	// should be gone and we can use constant at that time
 	if ds.NoSQL.PluginName != "cassandra" {
-		return fmt.Errorf("unknown NoSQL plugin name: %v", ds.NoSQL.PluginName)
+		return fmt.Errorf("unknown NoSQL plugin name: %q", ds.NoSQL.PluginName)
```

Comment:
I guess this is at the handler layer, but wouldn't %w be preferable due to the unwrapping properties?

Reply:
We don't have error to wrap here, I changed `%v` to `%q` to have  errors like `unknown NoSQL plugin name: ""` instead of `unknown NoSQL plugin name:`. 
Or do I miss something?

Reply:
both %w, %v and %q should all print out the error string as far as I know, but %w will keep the error as being available for higher layers to match off. But, as you point out, this is the handler layer, so this is probably not an issue.